operations/security_and_authentication/Change_NCN_Image_Root_Password_and_SSH_Keys.md

```diff
@@ -31,32 +47,65 @@ The Kubernetes image is used by the master and worker nodes.
    pit# mv -v *squashfs *kernel initrd* old
    ```
 
-1. Chroot into the image root
+1. Copy the generated public and private ssh keys for the root account into the image.
+
+   This example assumes that an RSA key was generated.
 
    ```bash
-   pit# chroot ./squashfs-root
+   pit# cp -p /root/.ssh/id_rsa /root/.ssh/id_rsa.pub squashfs-root/root/.ssh
    ```
 
-1. Change the password
+1. Add the public ssh key for the root account to `authorized_keys`.
+
+   This example assumes that an RSA key was generated so it adds the id_rsa.pub file to authorized_keys.
 
    ```bash
-   chroot-pit# passwd
+   pit# cat /root/.ssh/id_rsa.pub >> squashfs-root/root/.ssh/authorized_keys
+   pit# chmod 640 squashfs-root/root/.ssh/authorized_keys
    ```
 
-1. Replace the ssh keys
+1. Chroot into the image root
 
    ```bash
-   chroot-pit# cd root
+   pit# chroot ./squashfs-root
    ```
 
-1. Replace the default root public and private ssh keys with your own or generate a new pair with `ssh-keygen(1)`
+1. Change the password
 
    ```bash
-   chroot-pit# mknod /dev/urandom c 1 9
-   chroot-pit# ssh-keygen <options>
-   chroot-pit# rm /dev/urandom
+   chroot-pit# passwd
    ```
 
+1. If there are any other things to be changed in the image, they could also be done at this point.
+
+   For example, some sites may want to change the timezone.
+
+   1. (Optional) Set defalt timezone on management nodes.
```

Comment:
Since this is the only example of other things to change in the image, this top-level step can be removed. We already state in the above line that this is an example of changing the timezone. 

Maybe change line 81 to say...

For example, some sites may want to change the timezone. The following steps are an example of how to set the default timezone on management nodes.